packages/collector/test/tracing/messaging/bull/test.js

```diff
@@ -194,6 +240,8 @@ mochaSuiteFn('tracing/messaging/bull', function () {
           const withError = true;
           const urlWithParams = withError ? `${apiPath}&withError=true` : apiPath;
 
+          // TODO: all other bull tests also produce a huge number of spans
```

Comment:
Why did you move it here? 🤔 I suggested 232 (231 now). 

Pls reword as suggested:

> TODO: Investigate no. of spans https://jsw.ibm.com/browse/INSTA-15029

Reply:
Updated to the mentioned line number.
We are having this issue for many assertions. But one comment is enough I guess.